### PR TITLE
* Updated PowerTransformer proto to include powerTransformerInfo prop…

### PR DESCRIPTION
**Pull request by roberto-marquez**
…erty.

* Created PowerTransformerInfo proto.

Signed-off-by: Roberto Marquez <roberto.marquez@zepben.com>

### CODE REVIEW

**Comment by ryanjcoleman** [comment]
Task linked: [PROJ-228 Add PowerTransformerInfo to Evolve GRPC](https://app.clickup.com/t/6929263/PROJ-228)

File: docs/release.md [hunk 1]
```diff
@@ -42,7 +42,8 @@ a 1.0.0 release.
 * None.
 
 ##### Enhancements
-* None.
+* Updated PowerTransformer proto to include powerTransformerInfo property.
```
Comment (kgreav) [inline docs/release.md:45]: you didn't actually do this, so i assume the intention is to use the assetInfo field on PowerSystemResource?
if so you can leave out this line i think.

Comment (gcarwardine) [reply docs/release.md:45]: He had done this and I told him he didn't need it because of the assetInfo field on PSR. Obviously forgot to revert this.

**Review by gcarwardine — CHANGES_REQUESTED** [review]

File: docs/release.md (hunk 1 above)
Comment (roberto-marquez) [reply docs/release.md:45]: Yep, it is a leftover, I'll remove the release note.

**Review by kgreav — APPROVED** [review]

**Pull request merged**